src/styles/blocks/card.css

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  display: flex;
+  align-self: center;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__price-block {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__link {
+  width: 166px;
+  height: 40px;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  padding: 12px 55px;
+  text-decoration: none;
+  text-transform: uppercase;
+  display: inline-block;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.card__link:hover {
+  border: 1px solid #00acdc;
+  color: #00acdc;
+  background-color: #fff;
+  transition: color 0.5s;
```

Comment:
move `transition` to `.card__link`, and add `background-color` to it 😉

Reply:
done!